src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -99,35 +99,48 @@ public JsonWebToken(JObject header, JObject payload)
         public string Alg => Header.Value<string>(JwtHeaderParameterNames.Alg) ?? String.Empty;
 
         /// <summary>
-        /// Gets the list of 'audience' claim { aud, 'value' }.
+        /// Gets the list of 'aud' claim { aud, 'value' }.
         /// </summary>
-        /// <remarks>If the 'audience' claim is not found, enumeration will be empty.</remarks>
+        /// <remarks>If the 'aud' claim is not found, enumeration will be empty.</remarks>
         public IEnumerable<string> Audiences
         {
             get
             {
-                if (Payload != null)
+                if (Payload.GetValue(JwtRegisteredClaimNames.Aud) is JToken value)
                 {
-                    var value = Payload.GetValue(JwtRegisteredClaimNames.Aud);
-
                     if (value.Type is JTokenType.String)
                         return new List<string> { value.ToObject<string>() };
                     else if (value.Type is JTokenType.Array)
                         return value.ToObject<List<string>>();
                 }
 
-                return new List<string>();
+                return Enumerable.Empty<string>();
             }
         }
 
+        /// <summary>
+        /// Gets the original raw data of this instance when it was created.
```

Comment:
Gets the AuthenticationTag from the original...

Reply:
Done.